src/main/java/hudson/plugins/parameterizedtrigger/BuildTriggerConfig.java

```diff
@@ -412,9 +412,20 @@ private static void resolveProject(AbstractBuild build, SubProjectData subProjec
 			// don't trigger on this configuration
 		}
         return Collections.emptyList();
-	}
-
+    }
 
+    private void reportSchedulingError(@Nonnull Run<?, ?> run, @Nonnull Job<?, ?> jobToTrigger, @Nonnull BuildListener listener) {
+        // Do not print details to Build Listener, they have been reported previously in #canTriggerProject()
+        listener.error("Skipping " + jobToTrigger.getFullName() + "...");
+        if (LOGGER.isLoggable(Level.CONFIG)) {
```

Comment:
@jglick ADMIN already gets a WARNING here: https://github.com/jenkinsci/parameterized-trigger-plugin/blob/master/src/main/java/hudson/plugins/parameterizedtrigger/BuildTriggerConfig.java#L571-L582

This logging with Exception exists for extra diagnostics purpose only